File: 19주차/이연상/멀리 뛰기.py
# [PRG] 멀리 뛰기

def solution(n):
    d = [0] * (n+1)
    d[0] = 1
    d[1] = 2
    
    for i in range(2, n):
        d[i] = d[i - 1] + d[i - 2]
        
    return d[n - 1] % 1234567

File: 41주차/이연상/멀리 뛰기.java
// [PRG] 멀리 뛰기

class Solution {
    public long solution(int n) {
        long answer = 0;
        
        long d[] = new long[n + 1];
        d[0] = 1;
        d[1] = 2;
        for(int i = 2; i < n; i++) {
            d[i] = (d[i - 1] + d[i - 2]) % 1234567; 
        }
    
        return d[n - 1];
    }
}